projects/plugins/jetpack/modules/search/instant-search/components/path-breadcrumbs.scss

```diff
@@ -10,6 +10,10 @@
 
 .jetpack-instant-search__path-breadcrumb-link {
 	text-decoration: none;
+	-webkit-line-clamp: 1;
+    overflow: hidden;
```

Comment:
You might want to change the indentation to tabs, more information: `PCYsg-8-p2`

Reply:
ah, good reminder. Thank you!